src/PrestaShopBundle/Command/ModuleCommand.php

```diff
@@ -83,7 +83,7 @@ protected function init(InputInterface $input, OutputInterface $output)
         $this->translator = $this->getContainer()->get('translator');
         $this->input = $input;
         $this->output = $output;
-        require $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
+        require_once $this->getContainer()->get('kernel')->getRootDir() . '/../config/config.inc.php';
```

Comment:
I think this like is not needed anymore :thinking:

Reply:
removed